Examples/ObjectiveC/UserTrackingModesExample.m

```diff
@@ -3,23 +3,181 @@
 
 NSString *const MBXExampleUserTrackingModes = @"UserTrackingModesExample";
 
+#pragma mark - UIButton subclass
+
+// Subclass UIButton to create a custom user tracking mode button
+@interface UserLocationButton : UIButton
+@property (nonatomic) CAShapeLayer *arrow;
+@property (nonatomic) CGFloat buttonSize;
+@end
+
+@implementation UserLocationButton
+
+// Initializer to create the user tracking mode button
+- (instancetype)initWithButtonSize:(CGFloat)buttonSize {
+    if (self = [super init]) {
+        self.frame = CGRectMake(0, 0, buttonSize, buttonSize);
+        self.backgroundColor = [[UIColor whiteColor] colorWithAlphaComponent:0.9];
+        self.layer.cornerRadius = 4;
+        self.buttonSize = buttonSize;
+        
+        CAShapeLayer *arrow = [[CAShapeLayer alloc] init];
+        arrow.path = [self arrowPath];
+        arrow.lineWidth = 2;
+        arrow.lineJoin = kCALineJoinRound;
+        arrow.bounds = CGRectMake(0, 0, buttonSize / 2, buttonSize / 2);
+        arrow.position = CGPointMake(buttonSize / 2, buttonSize / 2);
+        arrow.shouldRasterize = YES;
+        arrow.rasterizationScale = [[UIScreen mainScreen] scale];
+        arrow.drawsAsynchronously = YES;
+        
+        self.arrow = arrow;
+        
+        // Update arrow for initial tracking mode
+        [self updateArrowForTrackingMode:MGLUserTrackingModeNone];
+        
+        [self.layer addSublayer:self.arrow];
+    }
+    
+    return self;
+}
+
+// Create a new bezier path to represent the tracking mode arrow,
+// making sure the arrow does not get drawn outside of the
+// frame size of the UIButton
```

Comment:
Nit: this is a sentence, please add a period at the end.

Reply:
Same as ☝️ https://github.com/mapbox/ios-sdk-examples/pull/151#discussion_r176216653 ❓ 🤔